app/forms/transfer/create_project_form.rb

```diff
@@ -52,4 +54,8 @@ def provisional_transfer_date=(hash)
   rescue TypeError, Date::Error, NegativeValueError
     @attributes_with_invalid_values << :provisional_transfer_date
   end
+
+  def check_incoming_trust_and_outgoing_trust
+    errors.add(:outgoing_trust_ukprn, I18n.t("errors.attributes.incoming_trust_ukprn.ukprns_must_not_match")) if incoming_trust_ukprn == outgoing_trust_ukprn
```

Comment:
If you add the error to `incoming_trust_ukprn` you can do:

`errors.add(:incoming_trust_ukprn, :ukprns_must_not_match) if incoming_trust_ukprn == outgoing_trust_ukprn`

Which also put the error on the field where it makes sense:

> Incoming trust must not be the same as the outgoing trust

Reply:
Yeah, this makes a lot more sense given the language used 👍